js/templates/userPage/store.html

```diff
@@ -40,15 +40,15 @@ <h1 class="txCtr">Loading...</h1>
           <div class="select2Small js-shipsToSelectDropdownContainer"></div>
         </div>
         <div class="flexVCent rowSm">
-          <input type="checkbox" class="margRSm js-filterFreeShipping" <% if (ob.filter.freeShipping) print('checked') %> />
-          <span class="clrE1 clrTOnEmph txTn txCtr upper padTn"><%= ob.polyT('userPage.store.freeShippingBanner') %></span>
+          <input type="checkbox" id="filterFreeShipping" class="margRSm margL0 js-filterFreeShipping" <% if (ob.filter.freeShipping) print('checked') %> />
```

Comment:
Since this is in a flex wrapper, it would be better to use a gutterHSm class on the parent than to monkeypatch things (to borrow a phrase from Cris S) with a margin override. 

I do see that if we do that the checkbox still has a left margin from the browser defaults, I've updated master with a fix.